core/src/replay_stage.rs

```diff
@@ -3206,14 +3210,25 @@ impl ReplayStage {
         };
 
         if let Some((bank, switch_fork_decision)) = selected_fork {
-            let (is_locked_out, vote_threshold, is_leader_slot, fork_weight) = {
+            let (
+                is_locked_out,
+                vote_threshold,
+                propagated_stake,
+                is_leader_slot,
+                fork_weight,
+                total_threshold_stake,
+                total_propagated_stake,
```

Comment:
`total_propagated_stake` is a little confusing here. Maybe just stick with `total_epoch_stake`